ironic-overcloud.yaml

```diff
@@ -1,5 +1,5 @@
 parameter_defaults:
```

Comment:
I didn't realize ironic-overcloud was a custom file! we should combine it with esi-custom